AnkiDroid/src/main/java/com/ichi2/anki/Previewer.java

```diff
@@ -58,8 +63,50 @@ protected void onCreate(Bundle savedInstanceState) {
         // Ensure navigation drawer can't be opened. Various actions in the drawer cause crashes.
         disableDrawerSwipe();
         startLoadingCollection();
+        initPreviewProgress();
     }
 
+
+    private void initPreviewProgress() {
+        LinearLayout progressLayout = findViewById(R.id.preview_progress_layout);
+        progressSeekBar = findViewById(R.id.preview_progress_seek_bar);
+        progressText = findViewById(R.id.preview_progress_text);
+        progressLayout.setVisibility(View.VISIBLE);
+        setSeekBar();
+        updateProgress();
+    }
+
+
+    private void setSeekBar() {
+        progressSeekBar.setMax(mCardList.length - 1);
+
+        progressSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+                if (fromUser) {
+                    mIndex = progress;
+                    updateProgress();
+                }
+            }
+
+
+            @Override
+            public void onStartTrackingTouch(SeekBar seekBar) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Document empty method body](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=4767237)

Reply:
Not used method but must be override..

Reply:
Sure, you just do something like this:
```suggestion
            public void onStartTrackingTouch(SeekBar seekBar) {
                // Mandatory override, but unused
```